docs/release-notes/workspace-namespace-binding.rst

```diff
@@ -0,0 +1,16 @@
+:orphan:
+
+**Breaking Changes**
+
+-  Cluster: The ``kubernetes_namespace`` field under the resource pool config is no longer
+   supported. Instead, the users can submit workloads to particular namespaces, by binding
+   workspaces to namespaces using the CLI or API.
+
+**New Features**
+
+-  Cluster: The ``namespace`` field under the Kubernetes Resource Manager config has been
+   deprecated, and a new field called ``default_namespace`` has been added. This field will be the
+   default namespace where Determined will deploy Pods and ConfigMaps, if the workspace that a
+   workload is submitted in, is not bound to a particular namespace. Currently, the master config
+   will accept either ``namespace`` or ``default_namespace`` fields, but if both are provided an
+   error will be thrown.
```

Comment:
:orphan:

**Breaking Changes**

- **Cluster:** The ``kubernetes_namespace`` field in the resource pool configuration is no longer supported. Users can now submit workloads to specific namespaces by binding workspaces to namespaces using the CLI or API.

**New Features**

- **Cluster:** The ``namespace`` field in the Kubernetes Resource Manager configuration has been deprecated. A new field, ``default_namespace``, has been introduced. This field serves as the default namespace for deploying Pods and ConfigMaps when the workspace associated with a workload is not bound to a specific namespace. The master configuration will accept either ``namespace`` or ``default_namespace`` fields; however, providing both fields will result in an error.